### PR TITLE
Improve README

### PR DESCRIPTION
**Pull request by kojix2**
* I tried to clarify the sentence.
* Avoid duplication of `provide` and `such as`
* Remove "TODO" notations that might discourage people from using Red Datasets.

### CODE REVIEW

**Comment by kojix2** [comment]
There are 3 commits. This is not what I want. This is because I repeat editing and checking in the browser window. For me, making lots of commits is the only way to improve Readmes. It's only after I commit that I can come up with another way to improve it...

File: README.md [hunk 1]
```diff
@@ -5,9 +5,9 @@
 
 ## Description
 
-Red Datasets provides classes that provide common datasets such as iris dataset.
+Red Datasets provides an easy way to fetch standard datasets commonly used in the data science community.
```
Comment (kou) [inline README.md]: I don't want to use "data science" here because we target more widely community.
We want normal Rubyists to use this library.

File: README.md [hunk 2]
```diff
@@ -5,9 +5,9 @@
...
 
-You can use datasets easily because you can access each dataset with multiple ways such as `#each` and Apache Arrow Record Batch.
+You can access these datasets in several formats, including Arrays, Numo::NArray, and ApacheArrow RecordBatch.
```
Comment (kou) [inline README.md]: Why did you remove `#each`?
This is the important interface of this rather than `Array`, Numo::NArray, Apache Arrow Record Batch and so on.

Comment (kou) [reply README.md]: Please don't change "Apache Arrow Record Batch" notation.
"ApacheArrow" (no space) is invalid.

File: README.md [hunk 3]
```diff
@@ -131,9 +131,12 @@ mnist.each do |record|
 end
 ```
 
-## NArray compatibility
+## Extensions
 
 * [red-datasets-numo-narray](https://github.com/red-data-tools/red-datasets-numo-narray)
+* [red-datasets-arrow](https://github.com/red-data-tools/red-datasets-arrow)
+* [red-datasets-daru](https://github.com/red-data-tools/red-datasets-daru)
+* [red-datasets-gdk-pixbuf](https://github.com/red-data-tools/red-datasets-gdk-pixbuf)
```
Comment (kou) [inline README.md]: Could you sort this list alphabetically?

Comment (kojix2) [reply README.md]: Got it.

File: README.md (hunk 1 above)
Comment (kojix2) [reply README.md]: Hmm.

File: README.md (hunk 2 above)
Comment (kojix2) [reply README.md]: I got it.

File: README.md (hunk 1 above)
Comment (kojix2) [reply README.md]: I'll think about it.

Comment (kojix2) [reply README.md]: Reverted. 
I think the idiom of the data science community is good. Because Ruby is also a language for data science and red-data-tools is a group that allows you to analyze data with Ruby. I think the data science community includes Ruby users. However, there is no big difference, so I will put it back.

File: README.md (hunk 2 above)
Comment (kojix2) [reply README.md]: I watched the video. https://www.youtube.com/watch?v=PI2M6qeHi_g
And I found that you think "each" is important.

Comment (kojix2) [reply README.md]: But I do not fully agree with that opinion. Yes, I admit that working with natural integration with the Ruby syntax is great. But what people really want to do is get a dataset with minimal effort. But, I do not have enough knowledge and time to investigate and tackle this difficult issue seriously, so I will withdraw...

File: README.md (hunk 3 above)
Comment (kojix2) [reply README.md:139]: done.

File: README.md (hunk 2 above)
Comment (kojix2) [reply README.md]: Reverted
The text I changed has a different meaning than before. This is not good.
